frontend/dockerfile/dockerfile_test.go

```diff
@@ -2079,7 +2194,8 @@ CMD ["test"]
 	require.NoError(t, err)
 	defer c.Close()
 
-	target := "docker.io/moby/cmdoverridetest:latest"
+	tag := integration.UnixOrWindows("latest", "windows")
```

Comment:
why do we need a specific tag for windows?

Reply:
Sure, I think we can just use the same tag since this is on the local mirror, will fix.